spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/lock/JdbcLockRegistry.java

```diff
@@ -151,7 +153,7 @@ public DistributedLock obtain(Object lockKey) {
 		}
 	}
 
-	private String pathFor(String input) {
+	private @Nullable String pathFor(@Nullable String input) {
 		return input == null ? null : UUIDConverter.getUUID(input).toString();
```

Comment:
I think `Objects.requireNonNull()` is better here instead.
The ` input` is never null, and result of the method must not be null.